framework/rendering/render_context.cpp

```diff
@@ -409,6 +415,19 @@ void RenderContext::end_frame(VkSemaphore semaphore)
 
 	// Frame is not active anymore
 	frame_active = false;
+	if (acquired_semaphore)
+	{
+		release_owned_semaphore(acquired_semaphore);
+		acquired_semaphore = VK_NULL_HANDLE;
+	}
+}
+
+VkSemaphore RenderContext::consume_acquired_semaphore()
+{
+	assert(frame_active && "Frame is not active, please call begin_frame");
```

Comment:
This error is triggered on some of the existing samples for me:

![image](https://user-images.githubusercontent.com/10283127/114220266-0e473f00-996c-11eb-8253-a1b5c4615f9d.png)

Tested with the MSAA sample, that shows above error in debug mode after the first few frames and then exists.

Reply:
Fixed.